### ISSUE TITLE
New Photon Voice update broke voice chat.

### ISSUE THREAD

**Issue by SteveTheAnimator** [fchb1239/PhotonVR#28]
Title

**Comment by fchb1239** [comment]
I don't have control over what Photon does with their packages, can you tell me exactly what's broken?

**Comment by SteveTheAnimator** [comment]
If you update both of the packages it breaks voice chat, if you only update pun it just breaks multiplayer.

**Comment by fchb1239** [comment]
How does it break? Any errors or bugs specifically?

**Comment by SteveTheAnimator** [comment]
The local player works fine, but nothing else works.

**Comment by Apache-Act1** [comment]
Yep im having this issue, I thought it was just me being dumb but then i saw this.

**Comment by SilvazWasTaken** [comment]
i also cant hear people speaking

**Comment by Apache-Act1** [comment]
For me i can only see the player (My own not other people)

**Comment by fchb1239** [comment]
Are there any errors in the log?

**Comment by Apache-Act1** [comment]
Nope.

**Comment by JuliaCodesShit** [comment]
Interesting

**Comment by SteveTheAnimator** [comment]
# How to fix it!
*Requirements*
A game with photon already installed

Put the old photon folder "Assets/Photon" into your new project, and bam!
Not the best method, but it works.

**Comment by SteveTheAnimator** [comment]
By the way, you need a one with the older photon and not the new photon.

**Comment by JuliaCodesShit** [comment]
Yea this was quite a obvious fix. I assumed that was known
